docs/resources/azurerm_management_group.md.erb

```diff
@@ -0,0 +1,274 @@
+---
+title: About the azurerm_management_group Resource
+platform: azure
+---
+
+# azurerm\_management\_group
+
+Use the `azurerm_management_group` InSpec audit resource to test properties related to a
+management group.
+
+<br />
+
+## Azure REST API version
+
+This resource interacts with version `2018-03-01-preview` of the Azure
+Management API. For more information see the [official Azure documentation](https://docs.microsoft.com/en-us/rest/api/resources/managementgroups/get).
+
+At the moment, there doesn't appear to be a way to select the version of the
+Azure API docs. If you notice a newer version being referenced in the official
+documentation please open an issue or submit a pull request using the updated
+version.
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      - name: inspec-azure
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+TODO: What should go here?
```

Comment:
No idea what do put here.

Reply:
Feel free to remove the Version section.